### PR TITLE
Updated some documentation for the OS class.

### PR DESCRIPTION
**Pull request by NathanWarden**

### CODE REVIEW

File: doc/classes/OS.xml [hunk 1]
```diff
@@ -476,6 +481,7 @@
 			<return type="bool">
 			</return>
 			<description>
+				Returns whether syncronizing the framerate to the monitor's refresh rate is enabled or not.
```
Comment (Noshyaar) [inline doc/classes/OS.xml]: syncronizing --> synchronizing

File: doc/classes/OS.xml [hunk 2]
```diff
@@ -688,6 +694,7 @@
 			<argument index="0" name="enable" type="bool">
 			</argument>
 			<description>
+				Sets whether or not to syncronize the framerate to the monitor's refresh rate.
```
Comment (Noshyaar) [inline doc/classes/OS.xml]: syncronize --> synchronize

Comment (NathanWarden) [reply doc/classes/OS.xml]: Thanks, fixed for both :)

File: doc/classes/OS.xml [hunk 3]
```diff
@@ -435,6 +437,9 @@
 			<return type="bool">
 			</return>
 			<description>
+				Returns true if the build is a debug build.
+				Returns true when running in the editor.
+				Returns false if the build is a release build.
```
Comment (cbscribe) [inline doc/classes/OS.xml]: We have standardized on writing [code]true[/code] and [code]false[/code].

File: doc/classes/OS.xml [hunk 4]
```diff
@@ -476,6 +481,7 @@
 			<return type="bool">
 			</return>
 			<description>
+				Returns whether synchronizing the framerate to the monitor's refresh rate is enabled or not.
```
Comment (cbscribe) [inline doc/classes/OS.xml]: Per the docs guidelines, this should follow the pattern:

If [code]true[/code] the framerate is synchronized...

**Review by NathanLovato — CHANGES_REQUESTED** [review]
Please add [code][/code] around true and false, and amend your commit. See the [writing guidelines](http://docs.godotengine.org/en/latest/community/contributing/docs_writing_guidelines.html)
Ping me (@NathanLovato) when you're done and I'll merge your PR :)

**Comment by NathanWarden** [comment]
@NathanLovato Is the correct format "If [code]true[/code] then the build is a debug build"?

**Comment by NathanWarden** [comment]
@NathanLovato Updated with code flags :)

**Review by NathanLovato — APPROVED** [review]

**Comment by NathanLovato** [comment]
The idea is to mark all code-related values with [code]. Then, we try to use action verbs to describe what the method does, which leads to your second example: `Returns...`
Thanks for your work! Merging now

**Comment by NathanWarden** [comment]
Thanks for letting me know, and thanks for your hard work with all the documentation. I also aspire to have a name just like yours one day :)

**Comment by NathanLovato** [comment]
You already have the same name :octocat:

**Pull request merged**